AmplifyPlugins/Core/AWSPluginsCoreTests/Model/Support/QueryPredicateGraphQLTests.swift

```diff
@@ -10,6 +10,7 @@ import XCTest
 @testable import AmplifyTestCommon
 @testable import AWSPluginsCore
 
+// swiftlint:disable type_body_length
 class QueryPredicateGraphQLTests: XCTestCase {
```

Comment:
If you are looking for a good place to test your QueryPredicate to GraphQL filter input, then you can update `GraphQLFilterCoverted.toJSON(predicate, options)` to take in model schema
`GraphQLFilterConverter.toJSON(predicate, modelSchema, options: [.prettyPrinted])`.